src/components/diff/PlainTextDiff.ts

```diff
@@ -105,9 +108,16 @@ export class PlainTextDiff extends Widget implements Git.Diff.IDiffWidget {
         this._challenger = await this._model.challenger.content();
       }
 
-      this.createDiffView(this._challenger, this._reference);
+      // Request base content only if base was provided in the model
+      this._base = await this._model.base?.content();
+
+      this.createDiffView(this._challenger, this._reference, this._base);
       this._challenger = null;
       this._reference = null;
+
+      // Set to null only if base was provided in the model
+      // else leave as undefined
+      this._base = !!this._model.base && null;
```

Comment:
No harm to set it to null?

```suggestion
      this._base = null;
```